analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CastShouldNotBeDuplicated.CSharp9.cs

```diff
@@ -24,10 +24,10 @@ public void Foo(Object x, Object y)
             switch (x)                            // Noncompliant [switch-st-0] {{Remove this cast and use the appropriate variable.}}
             {
                 case Fruit m:                     // Secondary [switch-st-1]
```

Comment:
Please add the ^^^^^ to a few cases to verify what is underlined when reporting the issue.